src/homework4/task5.py

```diff
@@ -0,0 +1,64 @@
+"""
+Языки
+
+Каждый из N школьников некоторой школы знает Mi языков.
+Определите, какие языки знают все школьники и языки, которые знает
+хотя бы один из школьников.
+"""
+
+
+# Входные данные
```

Comment:
inconsistent docstring - with """ at the beginning and # in the end. inaccurate.